ios/FluentUI/Button/Button.swift

```diff
@@ -11,13 +11,15 @@ import UIKit
 public enum ButtonStyle: Int, CaseIterable {
     case primaryFilled
     case primaryOutline
+    case dangerFilled
+    case dangerOutline
     case secondaryOutline
     case tertiaryOutline
     case borderless
```

Comment:
Should borderless also move up in the definition?

Reply:
I did that initially, but that also changed the order of the demo controller. So, I believe it's okay to leave this in the order of importance that we want to display. The switch cases I changed/created I made sure to put them in alphabetical order though.